src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -106,7 +110,7 @@ public String getCategoryLinkName() {
   }
 
   public String getUrlFromInternalLinkaAfterPageIsLoaded() {
```

Comment:
why are you waiting for body ? 
And I can't see usage of this method anywhere after, why are you modyfing it?